src/scoring/processor/expressions/mapResponse.js

```diff
@@ -81,39 +78,34 @@ var mapResponseProcessor = {
 
         //resolve the mapping
         if (variable.cardinality === 'single') {
-
             //find the map entry that matches with the value
-            mapResult = _.find(mapEntries, function(mapEntry) {
+            mapResult = _.find(mapEntries, function (mapEntry) {
                 if (variable.baseType === 'string' && mapEntry.attributes.caseSensitive === false) {
                     return _.isEqual(mapEntry.mapKey.toLowerCase(), variable.value.toLowerCase());
                 }
                 return _.isEqual(mapEntry.mapKey, variable.value);
             });
 
-            if (mapResult !== undefined) {
+            if (typeof mapResult !== 'undefined') {
                 result.value = parseFloat(mapResult.mapValue);
             }
-
         } else if (variable.cardinality === 'multiple' || variable.cardinality === 'ordered') {
-
             //get the entries that matches and sum their values
             mapResult = _(mapEntries)
-                .filter(function(mapEntry) {
-                    var found;
+                .filter(function (mapEntry) {
                     if (variable.baseType === 'string' && mapEntry.attributes.caseSensitive === false) {
-                        return _.contains(_.invoke(variable.value, 'toLowerCase'), mapEntry.mapKey.toLowerCase());
+                        return variable.value.map(val => val.toLowerCase()).includes(mapEntry.mapKey.toLowerCase());
                     }
                     if (_.isArray(mapEntry.mapKey)) {
-                        found = _.find(variable.value, mapEntry.mapKey);
-                        return found && found.length > 0;
+                        return variable.value.find(val => val.join(' ') === mapEntry.mapKey.join(' '));
                     }
-                    return _.contains(variable.value, mapEntry.mapKey);
+                    return variable.value && variable.value.includes(mapEntry.mapKey);
```

Comment:
```suggestion
                    return variable.value.includes?.(mapEntry.mapKey);
```

Reply:
`?` syntax leads to errors in tests when they are being run in console. Tests silently fail